library/clojure

```diff
@@ -2,72 +2,57 @@ Maintainers: Paul Lam <paul@quantisan.com> (@Quantisan),
              Wes Morgan <wesmorgan@icloud.com> (@cap10morgan)
 Architectures: amd64
 GitRepo: https://github.com/Quantisan/docker-clojure.git
-GitCommit: e96f6249cac065855a40a9f36ba88949e0a52d7d
+GitCommit: 09a243f78bde5ce2f5304b4d10a5d6b8152dc803
 
-Tags: openjdk-8, openjdk-8-lein, openjdk-8-lein-2.9.1, openjdk-8-stretch, openjdk-8-lein-stretch, openjdk-8-lein-2.9.1-stretch
-Directory: target/openjdk-8-stretch/lein
+Tags: latest
```

Comment:
It seems odd to only have `latest` on this image. Maybe it could have some more specific tags relating to `2.9.3`, `openjdk-11-slim`, or `buster` (and `tools-deps`?)?

Reply:
We’re being very intentional with that. See point #3 in the original list of changes. It’s a Clojure quirk that we’re trying to improve the UX around when people run one-liners with no tag specified.